### PR TITLE
Search form: handle wrong date format

### PR DESCRIPTION
**Pull request by pawelczak**
#445

### CODE REVIEW

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/judgmentSearch.js [hunk 1]
```diff
@@ -6,6 +6,67 @@
...
+    	showErrors: function(errorMap, errorList) {
+    		this.defaultShowErrors();
+    	},
+    	 highlight: function(element, errorClass, validClass) {
+    		 $(element).parent().parent().addClass("has-error");
+		 },
+		 unhighlight: function(element, errorClass, validClass) {
+			 $(element).parent().parent().removeClass("has-error");
+		 }
+    });
```
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/judgmentSearch.js]: indentation problem

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/dateFormat.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	
+	space.convert = function(text) {
+		var newText = "",
+			length = 0;
+		
+		text = text.replace(/-/g, '');
+		length = text.length;
+		
+		if (length > 2) {
+			text = insertAt(text, 2, "-");
```
Comment (madryk) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/dateFormat.js]: I guess you wanted to do operations on `newText` instead of `text`?

Comment (pawelczak) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/dateFormat.js]: Yeah, newText left from previous solution.

File: saos-webapp/src/main/webapp/WEB-INF/static/javascript/judgmentSearch.js (hunk 1 above)
Comment (pawelczak) [inline saos-webapp/src/main/webapp/WEB-INF/static/javascript/judgmentSearch.js]: :+1:

**Pull request closed without merging**